CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## 2.7.0-beta.1
+
+## Features ✨ and improvements 🏁
+
+* Enable preloading tiles for camera animation. ([#11328](https://github.com/mapbox/mapbox-gl-js/pull/11328))
+* Add perspective correction for non-rectangular image, canvas and video sources. ([#11292](https://github.com/mapbox/mapbox-gl-js/pull/11292))
+* Add function call chaining to function `map.setProjection`. ([#11279](https://github.com/mapbox/mapbox-gl-js/pull/11279)) (h/t [lpizzinidev](https://github.com/lpizzinidev))
+* Improve quality of transparent line layers by removing overlapping geometry artifacts. ([#11082](https://github.com/mapbox/mapbox-gl-js/pull/11082))
+* Improve performance of default markers. ([#11321](https://github.com/mapbox/mapbox-gl-js/pull/11321))
+* Add marker methods `setSnapToPixel` and `getSnapToPixel` to indicate rounding a marker to pixel. (h/t [@malekeym](https://github.com/malekeym)) ([#11167](https://github.com/mapbox/mapbox-gl-js/pull/11167)) (h/t [malekeym](https://github.com/malekeym))
+* Add a default aria-label for interactive markers for improved user accessibility. ([#11349](https://github.com/mapbox/mapbox-gl-js/pull/11349))
+* Allow users to set order of custom attribution. ([#11196](https://github.com/mapbox/mapbox-gl-js/pull/11196))
+
+### 🐞 Bug fixes
+
+* Fix canvas size to evaluate to expected value when applying the CSS transform property. ([#11310](https://github.com/mapbox/mapbox-gl-js/pull/11310))
+* Fix rendering of denormalized strings with diacritics. ([#11269](https://github.com/mapbox/mapbox-gl-js/pull/11269))
+* Fix a `getBounds` issue that sometimes causes returning invalid LngLat when zooming on a map with terrain. ([#11339](https://github.com/mapbox/mapbox-gl-js/pull/11339))
+* Fix a rare terrain flickering issue when using terrain with multiple vector data sources. ([#11346](https://github.com/mapbox/mapbox-gl-js/pull/11346))
+* Fix returning 404s with sparse terrain DEM tilesets. ([#11276](https://github.com/mapbox/mapbox-gl-js/pull/11276))
+* Fix Safari 15 warning: `TypeError: undefined is not an object (evaluating 't[12]')`. ([#11330](https://github.com/mapbox/mapbox-gl-js/pull/11330))
```

Comment:
Is there more context we can give to this one. In what circumstances did this get triggered? Is it a relatively rare bug? If so, maybe we should note it.

Reply:
It was a rare webkit bug solved by a different for loop - maybe unnecessary to include in changelog

Reply:
Yes you're right, sounds good skipping from what I recall of this issue! Seems very rare, platform and version specific.